python/xnvme-cy-bindings/xnvme/cython_bindings/tests/conftest.py

```diff
@@ -51,15 +51,28 @@ def dev(opts, request):
     opts.rdwr = 1
 
     device = xnvme.xnvme_dev_open(DEVICE_PATH, opts)
+    # if not conventional_namespace:
+    #     pytest.skip('Device must use conventional namespace')
```

Comment:
Is there any reason to keep this commented code?

Reply:
Not exactly, it served as a todo, but it's afaik not something that can just be fixed.